README.md

```diff
@@ -33,6 +36,7 @@ Thanks to all [contributors](https://github.com/thangchung/awesome-dotnet-core/g
   * [Logging](#logging)
   * [Mail](#mail)
   * [Mathematics](#mathematics)
+  * [Metrics](#metrics)
```

Comment:
This is the same. We have **Code Analysis and Metrics** so I don't think we need adding more **Metrics** section